requirements/edx/base.in

```diff
@@ -46,6 +46,7 @@ django-memcached-hashring
 django-method-override==0.1.0
 django-model-utils==3.0.0
 django-mptt>=0.8.6,<0.9
+django-mysql
```

Comment:
Is this new dependency still needed?

Reply:
Yes, I'm using this for the list of strings filed that stores the scopes on `ScopedApplication`.